src/eko/matching_conditions/__init__.py

```diff
@@ -80,11 +80,11 @@ def split_ad_to_evol_map(
                     # match the missing contribution from h+ and h-
                     m.update(
                         {
-                            f"{ihq}+.{ihq}+": ome_members["S_HH"],
-                            # f"S.{ihq}+": ome_members["S_qH"],
-                            f"g.{ihq}+": ome_members["S_gH"],
-                            f"{ihq}-.{ihq}-": ome_members["NS_HH"],
-                            # f"V.{ihq}-": ome_members["NS_qH"],
+                            f"{ihq}+.{ihq}+": ome_members[("H+", "H+")],
+                            # f"S.{ihq}+": ome_members[("S", "H+")],
+                            f"g.{ihq}+": ome_members[("g", "H+")],
+                            f"{ihq}-.{ihq}-": ome_members[("H-", "H-")],
+                            # f"V.{ihq}-": ome_members[("V", "H-")],
                         }
```

Comment:
Since we're actually changing, I'd use a lowercase `h` for the heavy quark, because in general is a placeholder for `c`, `b`, or `t`, so it's more consistent,